providers/implementations/ciphers/cipher_rc2.c

```diff
@@ -210,11 +210,13 @@ static int rc2_set_ctx_params(void *vctx, const OSSL_PARAM params[])
 
 CIPHER_DEFAULT_GETTABLE_CTX_PARAMS_START(rc2)
 OSSL_PARAM_size_t(OSSL_CIPHER_PARAM_RC2_KEYBITS, NULL),
+OSSL_PARAM_octet_string(OSSL_CIPHER_PARAM_ALG_ID, NULL, 0),
```

Comment:
Yes- this was definitely a bug, considering that this is the only cipher that uses this ID.